### DISCUSSION TITLE
Random nitpicks about the wiki

### DISCUSSION THREAD

**Discussion by 12beesinatrenchcoat** [DoobDev/Doob#193] [General]
I'm copying what I wrote in Discord here, because I'm a prick.

> > 4. Now, move the `servercount.py`, `stat.py`, and `music.py` to the NOT IN USE folder. (from the [wiki](https://github.com/DoobDev/Doob/wiki/Installing-and-working-on-Doob.))
>
> I seriously feel like there's a better way to do this...
> Would it be possible to, say... add a check to see if the necessary environment variable was there, and if not, don't load the cog? Something like that?

More notes:
> On Ubuntu (or similar OS like Pop-os), follow this guide: linuxize.com/post/how-to-install-python-3-8-on-ubuntu-18-04  

Python is a preinstalled package on Ubuntu (and likely also Pop!_OS (and also many other Linux operating systems)). On most Debian-based systems, `sudo apt install python3` suffices, if the user doesn't have it.

> 5. You should be able to start the bot!

Call me an idiot, but it took me a minute that I needed to execute `launcher.py`. That should probably be in there...

----
I also wonder if you should link the wiki under the contributing section of the README.

...sorry if this all sounds rude - I'm aware I'm nitpicking about all of this... (>_<)

**Comment by mmattbtw** [comment]
1. Issue moved to #197 
2. On my ubuntu system, I needed to install Python 3.8+, because Doob runs on that. Last time I remembered, Ubuntu/Pop!_OS didn't come with 3.8+
3. I'll add that 👍

**Reply by 12beesinatrenchcoat** [reply]
> Last time I remembered, Ubuntu/Pop!_OS didn't come with 3.8+

That's odd, but at the same time, I installed Ubuntu years ago, my memory might be a bit fuzzy on everything that was preinstalled (^^);

Thanks for the update!
